### PR TITLE
Update Brave Ads frequency capping study

### PR DESCRIPTION
**Pull request by tmancey**
Roll-out `exclude_ad_if_dismissed_within_time_window` and `exclude_ad_if_transferred_within_time_window` to 1.26.60 and above for tier 1 and 2 regions.

### CODE REVIEW

**Review by goodov — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Approved & Merged after deliberating with @brave/uplift-approvers.

**Pull request merged**
